### PR TITLE
Put login inside a Storyboard

### PR DESCRIPTION
**Pull request by Odrakir**
The main idea behind this is making the interface builder that holds the Login a `UIViewController` instead of the `UIView` that currently is.

The reason for this is because doing so we can use SafeArea and guarantee that it's backwards compatible with iOS 10 (as it only is for VCs, because it uses layout guides, which are only present in VCs).

The gain is not much for this particular screen, but for others we will be able to get rid of some magic numbers and also some code checking iOS version or iPhone model.

We can take care of other cases one at a time if this PR is approved.

### CODE REVIEW

File: Toggl.Daneel/ViewControllers/LoginViewController.cs [hunk 1]
```diff
@@ -12,23 +12,26 @@
 using Toggl.Foundation.MvvmCross.Helper;
 using Toggl.Foundation.MvvmCross.ViewModels;
 using UIKit;
+using System;
```
[the remark this answers is not in the document]
Comment (Odrakir) [reply Toggl.Daneel/ViewControllers/LoginViewController.cs]: I do need that for the `IntPtr` in the initializer.

**Comment by Odrakir** [comment]
I'm not sure what's going on with GH, but I deleted the file, and in fact it's not there: https://github.com/toggl/mobileapp/tree/login-to-viewcontroller/Toggl.Daneel/ViewControllers

**Comment by simonrozsival** [comment]
You somehow moved the file to `../Resources/ViewControllers/LoginViewController.xib` - https://github.com/toggl/mobileapp/blob/login-to-viewcontroller/Toggl.Daneel/Resources/ViewControllers/LoginViewController.xib - see the `Toggl.Daneel.csproj` file.

**Comment by Odrakir** [comment]
I decided to rename it to "Login.storyboard", as we agreed these storyboards main contain more than one VC if necessary (subviewcontrollers or whatever)

**Pull request merged**